### DISCUSSION TITLE
On QSO Data page, only show 'More QSOs' button if other QSOs are found in database

### DISCUSSION THREAD

**Discussion by gx1400** [magicbug/Cloudlog#2925] [Ideas / Feature Requests]
Improvement idea: Only show More QSOs if the operator in the QSO has already been logged.  

Sometimes I think I recognize a call, and want to confirm if I've already contacted them, and on what bands, but click the More QSOs button only to find a single entry.

![image of QSO data showing blue More QSOs button](https://github.com/magicbug/Cloudlog/assets/25909885/a20c9577-79e6-4cf0-8ae9-5f0603753367)

**Comment by magicbug** [comment]
Seems like a good suggestion

**Reply by gx1400** [reply]
Happens here right?

https://github.com/magicbug/Cloudlog/blob/0ff8192f9564d42c2115b3708ffae1d46239fcc6/application/views/view_log/qso.php#L393

**Reply by gx1400** [reply]
Sorry, I was looking to see if I could figure out how to implement and contribute, but I don't understand PHP well enough at the moment

@magicbug if this is worth implementing, do you generally convert it into an Issue? Or does it stay here in the discussion?
